reference_data/tests_only/simple_gap_model.json

```diff
@@ -5119,7 +5119,52 @@
       "init_params": {
         "representation": {
           "class_name": "SphericalInvariants",
-          "data": {},
+          "data": {
```

Comment:
This duplicates all the information already contained below (in `"init_params"`), which is not ideal

Reply:
issue has been solved by adapting cpp input parameters to python input parameters